README.md

```diff
@@ -2,9 +2,33 @@ Example Deployment
 -----------------
 This is just a simple example deployment for creating a Kubernetes Deployment on a Datica managed K8s cluster. We recommend using this as a baseline to expand off, as it is the most compliant deployment possible (using raw Load Balancers is NOT a good idea). Ingress is only needed if you want to make the service publicly available. Any value in braces is something you should replace, not something that Kubernetes understands.
 
+The ingress host_name should either be a DNS CNAME record pointing to the ingress-controller loadbalancer address, or the loadbalancer address itself for testing. The ingress-controller address can be found under the EXTERNAL-IP section in the output of the following command:
```

Comment:
Unless there is a specific thing customers would need to test using the LB address, I think we should strike that and always direct them to CNAME the DNS name.